torch/csrc/api/include/torch/fft.h

```diff
@@ -166,4 +166,65 @@ inline Tensor ihfft(const Tensor& self,
   return torch::fft_ihfft(self, n, dim, norm);
 }
 
+/// Returns Discrete Fourier Transform sample frequencies for signal size n.
```

Comment:
These comments are great but may need a small consistency update if the one-sentence descriptions in _torch_docs.py change.